maintenance/metric/jsonapi/jsonapi_test.go

```diff
@@ -12,29 +14,158 @@ import (
 	"github.com/pace/bricks/maintenance/metric"
 )
 
-func TestCaptureStatus(t *testing.T) {
-	rec := httptest.NewRecorder()
-	req := httptest.NewRequest("GET", "/test/1234567", nil)
+func TestMetric(t *testing.T) {
+	t.Run("capture metrics", func(t *testing.T) {
+		t.Run("api request", func(t *testing.T) {
+			rec := httptest.NewRecorder()
+			req := httptest.NewRequest("GET", "/test/1234567", nil)
 
-	handler := func(w http.ResponseWriter, r *http.Request) {
-		w = NewMetric("simple", "/test/{id}", w, r)
-		w.WriteHeader(204)
-	}
+			handler := func(w http.ResponseWriter, r *http.Request) {
+				w = NewMetric("simple", "/test/{id}", w, r)
+				w.WriteHeader(204)
+			}
 
-	handler(rec, req)
+			handler(rec, req)
+			req.Body.Close() // that's something the server does
 
-	resp := rec.Result()
-	defer resp.Body.Close()
+			resp := rec.Result()
+			defer resp.Body.Close()
 
-	if resp.StatusCode != 204 {
-		t.Errorf("Failed to return correct 204 response status, got: %v", resp.StatusCode)
-	}
+			if resp.StatusCode != 204 {
+				t.Errorf("Failed to return correct 204 response status, got: %v", resp.StatusCode)
+			}
+		})
+		t.Run("get metrics request", func(t *testing.T) {
+			rec := httptest.NewRecorder()
+			req := httptest.NewRequest("GET", "/metrics", nil)
```

Comment:
@threez You requested a change here, regarding a "small helper module", that I didn't understand. Can you elaborate?

Reply:
Moved to https://github.com/pace/bricks/issues/93